.coderabbit.yaml

```
@@ -0,0 +1,95 @@
+  exclude:
+    - "*.md"
+    - "package-lock.json"
+    - "yarn.lock"
+    - "*.generated.*"
+    - "dist/**"
+    - "build/**"
+    - "node_modules/**"
+    - "eslint.config.js"
+    - ".prettierrc"
+    - "tsconfig.json"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)